packages/native/src/useScrollToTop.tsx

```diff
@@ -82,20 +83,23 @@ export default function useScrollToTop(
           const scrollable = getScrollableNode(ref);
 
           if (isFocused && isFirst && scrollable && !e.defaultPrevented) {
-            if ('scrollToTop' in scrollable) {
```

Comment:
I'm not sure about this, since the hook is called `useScrollToTop`, it should prioritize the `scrollToTop` method if available.